src/main/java/domain/LottoConstants.java

```diff
@@ -0,0 +1,16 @@
+package domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoConstants {
+    public static final int MIN_NUMBER = 1;
+    public static final int MAX_NUMBER = 45;
+    public static final int LOTTO_LENGTH = 6;
+    public static final int LOTTO_PRICE = 1000;
+
+    public static final List<Integer> LOTTO_RANGES = IntStream.range(MIN_NUMBER, MAX_NUMBER)
+            .boxed()
+            .collect(Collectors.toList());
```

Comment:
```suggestion
    public static final List<Integer> LOTTO_RANGES = IntStream.rangeClose(MIN_NUMBER, MAX_NUMBER)
            .boxed()
            .collect(Collectors.toList());
```

range는 시작 범위를 포함하고, 끝 범위는 포함하지 않기 때문에 지금 코드에서는 1부터 44까지만 나옵니다~!
range 대신 rangeClose를 사용하면 끝 범위까지 포함되어서 출력 된답니다! 😄

Reply:
헉... 이건 몰랐네요. 반영해보겠습니다!